pkgs/shells/zsh/zsh-fzf-history-search/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib, stdenv, fetchFromGitHub, substituteAll, pkgs, bash, fzf }:
+
+stdenv.mkDerivation rec {
+  pname = "zsh-fzf-history-search";
+  version = "unstable-2023-03-08";
+
+  src = fetchFromGitHub {
+    owner = "joshskidmore";
+    repo = "zsh-fzf-history-search";
+    rev = "d1aae98ccd6ce153c97a5401d79fd36418cd2958";
+    hash = "sha256-4Dp2ehZLO83NhdBOKV0BhYFIvieaZPqiZZZtxsXWRaQ=";
+  };
+
+  strictDeps = true;
+  buildInputs = [ bash ];
+  propagatedBuildInputs = [ fzf ];
```

Comment:
```suggestion
```
Some shell plugins seem to leave external dependencies like fzf up to the user.
Otherwise, it should go to `buildInputs`.

e.g.
https://github.com/NixOS/nixpkgs/blob/nixos-unstable/pkgs/shells/zsh/zsh-fzf-tab/default.nix
https://github.com/NixOS/nixpkgs/blob/nixos-unstable/pkgs/shells/fish/plugins/fzf-fish.nix